### DISCUSSION TITLE
Change Overlay trigger behavior?

### DISCUSSION THREAD

**Discussion by N00Boj** [cymian/mouseless#75] [General]
I would find it more logical if the same key that's triggering the overlay would also cancel/hide it. I tried setting up the same key for both actions, to no avail (it stops working altogether).

Another thing I'd like to set up / request is using the show overlay trigger to reset the overlay (after having typed in the wrong cell, say). As I understand it, if I mistype, I have to cancel/hide the overlay first and then show it again to type the correct cell. Or am I missing an existing way to do this?

Cheers,
oj.

**Comment by cymian** [comment]
hi @N00Boj my apologies for the slow reply on this -- I will try to tend to the discussions tab more promptly :)

I fixed the first issue you mentioned (#77) in v0.2.2, and will be tracking status of the second issue in #18

please let me know if you have any more questions!
